### ISSUE TITLE
Configured webroot path not being used

### ISSUE THREAD

**Issue by sorcix** [hlandau/acme#85]
Hello,

After updating the `acmetool` binary I noticed that the webroot config file is no longer used. The directory structure doc tells me that the option is deprecated, `acmetool` logs pointed me to new location in `/var/run`.

Readme still says that webroot is a configurable path, but it looks like that's no longer the case? Please update documentation on how this should be used. Also: could you mention breaking changes like these on the release page, in the future? Thanks!

**Comment by hlandau** [comment]
You can configure custom webroot paths. acmetool also always uses `/var/run/acme/acme-challenge` as a standard location. It places challenge files in both that directory and any other directories you configure. If you configure no webroot paths, it still places challenges in `/var/run/acme/acme-challenge`. This is intended to be a standard location for challenge files and is also used by the redirector.

**Comment by sorcix** [comment]
Weird, this broke for me after updating from 0.34 to 0.38. The only related line in the logger output is:
```
20160125153554 [DEBUG] acme.responder: removing webroot file /var/run/acme/acme-challenge/HFWM2kIEqKcK7OtCoL9WtOmtZmCMBufyBauLhITFmhQ
```
Says nothing about my defined location.

Is there any other way to debug this issue?

**Comment by sorcix** [comment]
Could it be because I'm still using the `/var/lib/acme/conf/webroot-path` file?

**Comment by hlandau** [comment]
Can you pastebin the full log output? Redacting hostnames is OK.

**Comment by hlandau** [comment]
The output of `acmetool status` would also be helpful.

**Comment by bpereto** [comment]
+1
Same issue here

**Comment by hlandau** [comment]
@bpereto Please post debug log output, output of `acmetool status`, contents of `conf/target`.

**Comment by bpereto** [comment]
http://fpaste.org/314662/75698214/raw/

**Comment by bpereto** [comment]
runned `acmetool quickstart`, now its working.
It created the target file:

```
request:
  provider: https://acme-v01.api.letsencrypt.org/directory
  key:
    type: rsa
    rsa-size: 4096
  challenge:
    webroot-paths:
    - /var/www/html/le/.well-known/acme-challenge
```

**Comment by hlandau** [comment]
Bug confirmed, cause under investigation. But yes, a workaround is to move webroot paths to conf/target. The legacy configuration file for webroot paths is not being used for some reason.

**Comment by hlandau** [comment]
Fix in v0.0.41.
